td/tasks.py

```diff
@@ -1,32 +1,68 @@
 from __future__ import absolute_import
 
-from django.db import connection
-
 from celery import task
+from django.db import connection
 from pinax.eventlog.models import log
 
 from td.imports.models import (
-    EthnologueLanguageCode,
-    EthnologueCountryCode,
-    SIL_ISO_639_3,
-    WikipediaISOLanguage,
-    WikipediaISOCountry,
-    IMBPeopleGroup
-)
-
+    EthnologueLanguageCode, EthnologueCountryCode, SIL_ISO_639_3,
```

Comment:
Also, while not in the contribution guidelines (I need to update them), let's keep this a single list that is alpha sorted, as it's easier to read in a single list.